Gemfile

```diff
@@ -23,4 +23,8 @@ group(:dev) do
   gem 'guard'
   gem 'guard-rspec'
   gem 'ruby_gntp'
+  gem 'chefspec'
+  gem 'rubocop'
+  gem 'knife-spork'
+  gem 'kitchen-vagrant'
```

Comment:
Did we change it to avoid the magical behavior?



> On Sep 11, 2015, at 7:01 PM, Matt Wrock <notifications@github.com> wrote:
> 
> In Gemfile:
> 
> > @@ -23,4 +23,8 @@ group(:dev) do
> >    gem 'guard'
> >    gem 'guard-rspec'
> >    gem 'ruby_gntp'
> > +  gem 'chefspec'
> > +  gem 'rubocop'
> > +  gem 'knife-spork'
> > +  gem 'kitchen-vagrant'
> yeah. @lamont-granquist is right. :development is the magical one and I think that sounds like a good idea to stick to that convention.
> 
> —
> Reply to this email directly or view it on GitHub.
>